docs/supported_apis/series_supported.rst

```diff
@@ -12,13 +12,16 @@ the method in the left column. ``Y`` stands for yes, ``N`` stands for no, ``P``
 for partial (meaning some parameters may not be supported yet), and ``D`` stands for
 default to pandas. To learn more about the implementations that default to pandas, see
 the related section on :doc:`Defaulting to pandas </supported_apis/index>`.
+**Currently second column reflects implementation status for ``Ray`` and ``Dask`` engines. By default support of a method
+in the ``Omnisci`` engine could be treated as ``D`` unless ``Notes`` column contains additional information. Similarly,
+by default ``Notes`` contains information about ``Ray`` and ``Dask`` engines unless ``Omnisci`` explicitly mentioned.**
 
 +-----------------------------+---------------------------------+----------------------------------------------------+

```

Comment:
Comments left for `docs/supported_apis/dataframe_supported.rst` are applicable for this file too.